### PR TITLE
[fix] Error moving between grids while in FM imaging mode

### PR DESCRIPTION
**Pull request by patrickcleeve2**
- Fixes an issues where users couldn't swap between grids while in FM imaging mode. (https://support.delmic.com/scp/tickets.php?id=1453#reply)

### CODE REVIEW

File: src/odemis/acq/move.py [hunk 1]
```diff
@@ -370,16 +373,19 @@ def getTargetPosition(self, target_pos_lbl: int) -> Dict[str, float]:
...
                     sem_grid1_pos = stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]]
+                    sem_grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
                     fm_target_pos = self._transformFromSEMToMeteor(sem_grid1_pos)
                 elif current_position == SEM_IMAGING:
                     fm_target_pos = self._transformFromSEMToMeteor(self.stage.position.value)
                 end_pos = fm_target_pos
         elif current_position == FM_IMAGING:
             if target_pos_lbl == GRID_1:
                 sem_grid1_pos = stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]]
+                sem_grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
```
Comment (K4rishma) [inline src/odemis/acq/move.py:384]: Are these changes only required for TFS and not for Tescan and Zeiss?

Comment (patrickcleeve2) [reply src/odemis/acq/move.py:384]: Yes, the check is only supported in TFS (for now)

Comment (K4rishma) [reply src/odemis/acq/move.py:384]: This support is already there for Tescan and Zeiss, my bad. The order followed there is updating the angle and then the grid centre coordinates, I think both give the same results. But can we make it consistent for all i.e. TFS, Tescan and zeiss regarding the above update?

**Review by K4rishma — APPROVED** [review]

**Review by tepals — APPROVED** [review]

File: src/odemis/acq/move.py (hunk 1 above)
Comment (patrickcleeve2) [reply src/odemis/acq/move.py:384]: Yep, you are correct its a different order for each manufacturer. Will update

**Pull request merged**